src/models/propulsion/FGRocket.cpp

```diff
@@ -88,7 +88,7 @@ FGRocket::FGRocket(FGFDMExec* exec, Element *el, int engine_number, struct Input
 
   auto PropertyManager = exec->GetPropertyManager();
   bindmodel(PropertyManager.get()); // Bind model properties first, since they might be needed in functions.
-
+  FDMExec->SetPropertyValue("propulsion/engine/rocket/start-state", FDMExec->GetPropertyValue("propulsion/engine/rocket/initial_state"));
```

Comment:
Same thing, not liking the fact that if this property doesn't exist, it will crash JSBSim.